src/assets/templates/typescript/src/steps/users/user-field-equals.ts

```diff
@@ -52,10 +53,11 @@ export class UserFieldEqualsStep extends BaseStep implements StepInterface {
   }, {
     field: 'operator',
     type: FieldDefinition.Type.STRING,
-    description: 'Check Logic (be, not be, contain, not contain, be greater than, or be less than)',
+    description: 'Check Logic (be, not be, contain, not contain, be greater than, be less than, be set, or not be set)',
   }, {
     field: 'expectedValue',
     type: FieldDefinition.Type.ANYSCALAR,
+    optionality: FieldDefinition.Optionality.OPTIONAL,
```

Comment:
@amcpanaligan although the test run output isn't visible in the build, I suspect it's related to the automated test fail on this optionality.